src/main/resources/application.yaml

```diff
@@ -731,6 +755,30 @@ notification:
             emailId: 97c58e23-c11f-40b3-b981-2d4cfa38b8fd
             smsId: bb3df0ea-8259-43c4-95de-9eef96206575
   welsh:
+    actionFurtherEvidence:
+      docmosisId: TB-SCS-LET-ENG-representation-letter.docx
```

Comment:
Shouldn't there be a welsh template here?

Reply:
Fixed